.gitlab-ci.yml

```diff
@@ -48,6 +48,10 @@ workflow:
       variables:
         DEPLOY_TAG: $CI_COMMIT_TAG
 
+    - if: $CI_COMMIT_REF_NAME =~ /feature/
+      variables:
+        DEPLOY_TAG: "9.x-dev"
```

Comment:
Without this, the image defaults to `9.x-latest` which causes the catch in build to fail because deploying to latest branches requires manual flags to be set.